### PR TITLE
Adding exercises to error-handling chapter

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: py-rse/errors.Rmd [hunk 1]
```diff
@@ -28,14 +28,16 @@ for denom in [-5, 0, 5]:
...
 If something *does* happen inside the `try`,
 the program jumps immediately to the `except`.
 This is why the `print` statement inside the `try` doesn't run when `denom` is 0:
 as soon as Python tries to calculate `1/denom`,
 it skips directly to the error message.
 
-FIXME: diagram of exception handling control flow
+```{r py-rse-errors-control-flow, echo=FALSE, fig.cap="Exception Control Flow"}
+knitr::include_graphics("figures/FIXME.png")
+```
```
Comment (DamienIrving) [inline py-rse/errors.Rmd:40]: Can you provide a sketch of this figure?

File: py-rse/errors.Rmd [hunk 2]
```diff
@@ -564,23 +566,37 @@ because then they can set it up the way they want with very little work.
...
--   FIXME: use `logging_tree.printout()` to display logging configuration.
+1.  Modify your solution to the previous exercise
+    to put your error message in a catalog
+    as described in Section \@ref(py-rse-errors-messages).
+2.  Add messages in a second language.
+    (Use Google Translate if necessary).
+3.  Add a command-line flag to allow users to select the language they want to use.
 
-### Log to standard error as well as a file {#py-rse-errors-ex-log-stderr}
+### Set the logging level {#py-rse-errors-ex-set-level}
```
Comment (DamienIrving) [inline py-rse/errors.Rmd:588]: How does this exercise fit with the fact that in #343 logging is now added to `collate.py` during the lesson?

File: py-rse/errors.Rmd (hunk 1 above)
Comment (gvwilson) [reply py-rse/errors.Rmd:42]: Added.

**Comment by gvwilson** [comment]
@DamienIrving comments addressed and rough diagram added - i hope this one is ready enough for merging.

**Pull request merged**